buckw

```diff
@@ -128,11 +128,11 @@ runOkBuck ( ) {
     if [[ ! -z "$INSTALLED_WATCHMAN" ]]; then
         getToClean | jsonq '"\n".join(obj["files"])' | xargs rm
         info "DELETED OLD BUCK FILES"
-        EXTRA_ARGS="-xokbuckClean"
+        CLEAN_ARG="-xokbuckClean"
```

Comment:
Can we call this `OKBUCK_ARGS`

We have some changes coming for incremental okbuck and the name would be more appropriate